### ISSUE TITLE
Flip Image horizontally

### ISSUE THREAD

**Issue by vanniktech** [guoyingtao/Mantis#185]
Currently, I'm doing:

```swift
extension CropViewController : ImageEditorDelegate {
    public func flipImageHorizontally() {
        image = image.withHorizontallyFlippedOrientation()
    }
}
```

Is that how we should do it or do we want to provide some API for it?

**Comment by guoyingtao** [comment]
Currently I don't have any plan to provide APIs like that because Mantis is kind of WYSIWYG but you are welcome to do this in your own code.
I do want to support mirror mode which was mentioned in #47 in the future which may actually do the same thing you mentioned but it needs more UI works as well.

**Comment by guoyingtao** [comment]
@vanniktech 
Just let you know I restored showRotationDial setting in CropViewConfig because I think it is easier and straightforward for a user to set it.

**Comment by vanniktech** [comment]
Huh my `flipImageHorizontally ` does not really work:

<img width="300" alt="Screen Shot 2022-07-27 at 18 18 26" src="https://user-images.githubusercontent.com/5759366/181298090-93147646-b1ce-4e2c-bea0-a56aa8f8504f.png">

The cropping box stays and shows the wrong thing:

<img width="300" alt="Screen Shot 2022-07-27 at 18 18 51" src="https://user-images.githubusercontent.com/5759366/181298176-e7e00549-c624-4a6b-b172-50914048af74.png">

**Comment by guoyingtao** [comment]
I thought you only wanted to output the flipped image.

If you want to do the entire interactions with flipped image, it may need more work than you thought. Also it may need Mantis to support this feature inside the library.

**Comment by vanniktech** [comment]
> I thought you only wanted to output the flipped image.

Nope. 

> Also it may need Mantis to support this feature inside the library.

Yes. In the very beginning I only tested it out when flipping without doing any interactions (rotate, zoom in etc)

**Comment by guoyingtao** [comment]
@vanniktech 
Finally I got a chance to make Mantis support flipping image.
Please check this PR #200 to see if it works for you

For demo example, you can choose `Clockwise rotation` button, then you will see `flip horizontally` and `flip vertically` button
<img width="386" alt="image" src="https://user-images.githubusercontent.com/26723384/186811770-3e20a39e-b026-4a70-a964-1188724d8a47.png">

<img width="407" alt="image" src="https://user-images.githubusercontent.com/26723384/186811834-597e0e2c-6b39-482a-b8e1-c3a757b34b69.png">

**Comment by guoyingtao** [comment]
This is a simple demo
![Simulator Screen Recording - iPhone 13 Pro Max - 2022-08-25 at 23 33 49](https://user-images.githubusercontent.com/26723384/186812000-260d5361-f4eb-4d97-b91b-3f1984821165.gif)

**Comment by vanniktech** [comment]
Amazing, will check it this weekend!

**Comment by guoyingtao** [comment]
I closed this issue since Mantis 2.1.0 supports flipping Image now.
